enterprise/app/code/code.tsx

```diff
@@ -55,6 +56,10 @@ interface State {
 }
 
 const LOCAL_STORAGE_STATE_KEY = "code-state-v1";
+var repoLoaded: (value: any) => void;
+var repoLoadedPromise = new Promise<any>(function (resolve) {
+  repoLoaded = resolve;
+});
```

Comment:
nit: can you make these fields of CodeComponent? otherwise I think there could be problems when switching between this page and other pages

Reply:
Removed them